lighthouse/src/main.rs

```diff
@@ -324,6 +324,30 @@ fn main() {
                 .takes_value(true)
                 .global(true)
         )
+        .arg(
+            Arg::with_name("genesis-state-url")
```

Comment:
There are no CLI tests for these new flags

Reply:
Good catch, I added some here: https://github.com/sigp/lighthouse/pull/4653/commits/9bdc2433320d02ce8f8db34fcd9dfbfddd3d7f0c

I wanted to test the behaviour with `--checkpoint-sync-url` however our test don't support it (they try to download a checkpoint state). I don't have the time to go down that rabbit hole at present.